wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/XmlHttpTestResponseSchemaValidation.wiki

```diff
@@ -0,0 +1,54 @@
+---
+Test
```

Comment:
This test seems more appropriate for the HsacAcceptanceTests, so we run it always.

A single example (with an external XML file?) is also nice, but ensuring it works for every build....